lib/ecto/migration.ex

```diff
@@ -524,32 +527,33 @@ defmodule Ecto.Migration do
 
   ## Index types
 
-  PostgreSQL supports several index types like B-tree, Hash or GiST. When
+  PostgreSQL supports several index types like B-tree, Hash, and GiST. When
```

Comment:
Should we also mention `GIN`? What about also mentioning MySQL?

Reply:
Or, what do you think about?

> ## Index types
>
> When creating an index, the index type can be specified with the `:using` option. The `:using` option can be an atom or a string; and its value is passed to the `USING` clause as is.
>
> For example, PostgreSQL supports several index types like B-tree (the default), Hash, GIN, or GiST. More information on index types can be found in the [PostgreSQL docs](http://www.postgresql.org/docs/9.4/static/indexes-types.html).

I think in that case we don't need to mention MySQL 🙂

Reply:
Yeah, that seems like a good compromise. Used your phrasing 🙂